Baekjoon/10866.py

```diff
@@ -0,0 +1,54 @@
+'''
+문제 링크:https://www.acmicpc.net/problem/10866
+카테고리:덱
+문제해설:덱(Deque)를 구현한 다음, 입력으로 주어지는 명령을 처리
+'''
+from collections import deque
+import sys
+
+n=int(input())
+input=sys.stdin.readline
+
+d=deque([])
+
+for i in range(n):
+  commend=input().split()
+
+  if commend=='push_front':
+    d.appendleft()
+
+  if commend=='push_back':
+    d.append()
+
+  if commend=='pop_front':
+    if d:
+      print(d.popleft())
+    else:
+      print(-1)
+
+  if commend=='pop_back':
+    if d:
+      print(d.pop())
+    else:
+      print(-1)
+
+  if commend=='size':
+    print(len(d))
+
+  if commend=='empty':
+    if d:
+      print(0)
+    else:
+      print(1)
+
+  if commend=='front':
+    if d:
+      print(d[0])
+    else:
+      print(-1)
+
+  if commend=='back':
+    if d:
+      print(d[n-1])
+    else:
+      print(-1)
```

Comment:
```suggestion
  if commend=='back':
    if d:
      print(d[-1])
    else:
      print(-1)
```

n 이 정의되지 않아서 오류가 발생할 것 같습니다. deque의 길이를 의미한다면 `d[len(d)-1]`를 사용할 수 있겠지만 파이썬이니 `d[-1]`을 이용하여 마지막 요소에 접근할 수 있어요~!